api/src/database/seeds/05-activity.yaml

```diff
@@ -20,7 +20,7 @@ columns:
   user_agent:
     type: string
     length: 255
-    nullabel: false
+    nullable: false
```

Comment:
scary

Reply:
Granted this is definitely a typo, I'm wondering will user agent ever be `null`? Thinking out loud here.

Reply:
If you mean browser user agents, I'd imagine only if an extension is doing some weird hacking. I think I may have caused this behavior in Gecko twenty years ago. It wouldn't be a good idea as it's fairly likely to break everything.

Reply:
Yeah not that I'd want to not correct this, just want to avoid it potentially breaking anything. However since this _should_ only apply to new installs moving forward, it should be fine. 

Though I do wonder should we add a migration for existing projects... cc @rijkvanzanten

Reply:
We can't change the seeds, so lets revert this change 👍🏻